iOS/Swift/SWIFTLINT.md

```diff
@@ -0,0 +1,25 @@
+# Swiftlint
+
+A very good project on github is [Swiftlint](https://github.com/realm/SwiftLint)
+
+Install this in your projects to keep them clean. More info on Swiftlint github.
+
+As an example you can use a `.swiftlint.yml` file to customize the rules you want and don't want.
+
+```yml
+excluded:
+  - ./Pods
+  - ./<Project>Tests
+
+disabled_rules:
```

Comment:
Ik vind line legth wel een nuttige al mag het wat langer. body_length zou ik enkel uit willen hebben in testen. file_length zou ik zeker aan laten.